src/main/scala/shine/DPIA/showDPIA.scala

```diff
@@ -75,6 +75,8 @@ case class showDPIA(showTypes : ShowTypes = Off) {
     case Cast(_, dt2, e) => s"cast ${showKindedType(DataKind, dt2)} ${showPhrase(e)}"
     case RotateValues(_,_,_,wrt,in) => s"rotate ${showPhrase(in)}\n${tab(showPhrase(wrt))}"
     case f : ReduceSeq => s"reduce ${showPhrase(f.array)} ${showPhrase(f.init)}\n${tab(showPhrase(f.f))}"
+    case kc: shine.OpenCL.primitives.imperative.KernelCallCmd =>
+      s"${kc.name}<<<${kc.localSize}, ${kc.globalSize}>>>(${kc.args}, ${kc.output})"
```

Comment:
Is this using CUDA syntax for the OpenCL kernel call?

Can we maybe avoid this (possible confusion) even when this is just a debug output?

Reply:
I was not sure what other syntax to use, I don't think that there is an OpenCL equivalent for this (it is higher level than `clEnqueueNDRangeKernel`). Or is there a SYCL equivalent?

Reply:
I don't think there is a convenient SYCL syntax either. I just think that using the CUDA syntax can be quite confusing. How about just writing something like:
```
s"oclKernelLaunch(${kc.name})(${kc.localSize}, ${kc.globalSize})(${kc.args}, ${kc.output})"
```

Reply:
I've done something like that and added some more OpenCL print cases as well. Here is an example:
```
// before HostManagedBuffers.insert
new (exp[n1.i32, read] x acc[n1.i32]) in 
  λ(x101).
    (/* mapSeq */;
    for n1
      λ(x109).
        (((proj2 (x101))[x109]) = (cast (i32) 2) + ((e2)[x109]));
    output = oclLaunchKernel(k0, LocalSize(2,1,1), GlobalSize(((n1) / (2)),1,1))(NatAsIntData(n1), proj1 (x101)))

// after analyzeAndInsertHostExecution
new (exp[n1.i32, read] x acc[n1.i32]) in 
  λ(x101).
    (oclHostExecution (e2 HOST_READ, x101 HOST_WRITE) in
      /* mapSeq */;
      for n1
        λ(x109).
          (((proj2 (x101))[x109]) = (cast (i32) 2) + ((e2)[x109]));
    output = oclLaunchKernel(k0, LocalSize(2,1,1), GlobalSize(((n1) / (2)),1,1))(NatAsIntData(n1), proj1 (x101)))

// after insertManagedBuffers
newManagedBuffer (exp[managed[n1.i32], read] x acc[managed[n1.i32]] HOST_WRITE | DEVICE_READ) in 
  λ(mx101).
    (oclHostExecution (e2 HOST_READ, x101 HOST_WRITE) in
      /* mapSeq */;
      for n1
        λ(x109).
          (((proj2 (x101))[x109]) = (cast (i32) 2) + ((e2)[x109]));
    moutput = oclLaunchKernel(k0, LocalSize(2,1,1), GlobalSize(((n1) / (2)),1,1))(NatAsIntData(n1), proj1 (mx101)))
```